src/libvirtApi/common.js

```diff
@@ -499,10 +499,10 @@ export function enableLibvirt({ enable, serviceName }) {
     return enable ? libvirtService.enable() : libvirtService.disable();
 }
 
-export function getApiData({ connectionName, libvirtServiceName }) {
+export function getApiData({ connectionName }) {
     if (connectionName) {
         dbusClient(connectionName);
-        startEventMonitor({ connectionName, libvirtServiceName });
+        startEventMonitor({ connectionName });
```

Comment:
The `startEventMonitor` function still mentions `@param {String} libvirtServicename` in it's JSDoc

Reply:
Thanks :) Addressed.